build-in-container/run-container.sh

```diff
@@ -17,15 +17,15 @@ run_build_container() {
         ${mount_pts} \
         --privileged \
         --cap-add SYS_ADMIN \
-        mcr.microsoft.com/mariner-container-build:2.0 /mariner/scripts/build-mariner.sh
+        mcr.microsoft.com/mariner-container-build:2.0 /mariner/scripts/build-mariner.sh $container_args
```

Comment:
Since this isn't a user-facing component should it be in the `./scripts` dir?

Reply:
the scripts inside /scripts are copied into the docker. The scripts outside /scripts (3) are responsible for interacting with the user, and creating and running containers. Only one out of these three scripts is actually user-facing.